apps/services/auth/admin-api/infra/auth-admin-api.ts

```diff
@@ -59,6 +59,12 @@ export const serviceSetup = (): ServiceBuilder<'services-auth-admin-api'> => {
         prod: 'IS/GOV/5402696029/Skatturinn/ft-v1',
       },
       COMPANY_REGISTRY_REDIS_NODES: REDIS_NODE_CONFIG,
+      SYSLUMENN_HOST: {
+        dev: 'https://api.syslumenn.is/staging',
+        staging: 'https://api.syslumenn.is/staging',
+        prod: 'https://api.syslumenn.is',
+      },
+      SYSLUMENN_TIMEOUT: '10000',
```

Comment:
10 second timeout is quite long, is it really necessary to have it set this high?

Reply:
Good point, decreased to 3 s.